core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/datastore/oplock/DatastoreLockIndexEntry.java

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright 2019 B2i Healthcare Pte Ltd, http://b2i.sg
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.b2international.snowowl.datastore.oplock;
+
+import java.io.Serializable;
+import java.util.Collection;
+import java.util.Objects;
+
+import com.b2international.index.Doc;
+import com.b2international.index.mapping.DocumentMapping;
+import com.b2international.index.query.Expression;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Preconditions;
+
+/**
+ * @since 7.1.0
+ */
+@Doc(type = "lock")
+@JsonDeserialize(builder=DatastoreLockIndexEntry.Builder.class)
+public class DatastoreLockIndexEntry implements Serializable {
+	
+	private static final long serialVersionUID = 1L;
+	
+	public static final class Fields {
+		public static final String ID = "id";
+		public static final String USER_ID = "userId";
+		public static final String DESCRIPTION = "description";
+		public static final String PARENT_DESCRIPTION = "parentDescription";
+		public static final String REPOSITORYUUID= "repositoryUuid";
+		public static final String BRANCHPATH = "branchPath";
+	}
+	
+	public static class Expressions {
+		public static Expression id(final String id) {
+			return DocumentMapping.matchId(id);
+		}
+		
+		public static Expression ids(final Collection<String> ids) {
+			return com.b2international.index.query.Expressions.matchAny(DocumentMapping._ID, ids);
```

Comment:
Methods from `com.b2international.index.query.Expressions`  can be imported via static import, so you can simplify lines like these to `matchAny(DocumentMapping._ID, ids);`,